games-vue-client/src/components/games/Hanabi.vue

```diff
@@ -112,9 +115,22 @@ export default {
       this.onAction(action, index);
     }
   },
+  watch: {
+    deckEmpty: function (val) {
+      this.snackbar = val
+    }
+  },
+  data: () => ({
+    snackbar: false,
+    timeout: 5000,
+    text: 'Last round',
```

Comment:
I think the `timeout` attribute is unused. Also `text` could be renamed to `snackbarText`. Other than that, it looks good.

Reply:
I swapped to snackbarText.
Timeout is needed to specify for how long the message appears before disappearing.
Without a timeout, it will be visible forever